tbx/core/blocks.py

```diff
@@ -274,6 +286,11 @@ class StoryBlock(StreamBlock):
         group="Media",
     )
     video_block = VideoBlock(group="Media")
+    client_logo_block = ClientLogoBlock(
+        icon="openquote",
```

Comment:
I don't think the client logo block is available on every page - its one of the service page blocks so should be part of a new streamfield definition that expands this one, and then adds the new service page blocks in.

It might be good to name this 'partners' for consistency with the fieldspec.